test/luatest_helpers/vtest.lua

```diff
@@ -218,15 +222,22 @@ local function cluster_new(g, cfg)
             local user = box.session.user()
             box.session.su('admin')
 
+            local grant_range = cfg.test_user_grant_range
+            ivtest.clear_test_cfg_options(cfg)
             ivshard.storage.cfg(cfg, box.info.uuid)
-            box.schema.user.grant('storage', 'super')
+
+            if grant_range ~= nil then
```

Comment:
Nit: Maybe we should grant `super` by default, when `grant_range` is not specified? This way you won't have to change `router_test`

Reply:
I intentionally didn't want to do that. It should help to catch cases when we somehow miss adding a needed privilege automatically, but it won't be breaking the tests because they would always grant `super`.